config/locales/event.en.yml

```
@@ -58,5 +57,5 @@ en:
     mark_as_require_data_review: Requires data review
     mark_as_require_response: Requires response
     reset_to_initial_state: Case type changed, and status reset
-    reopen: Reopen
-    reopen__desc: Case reopened
+    mark_as_legal_proceedings_ongoing: Mark as ongoing legal case
+    mark_as_legal_proceedings_ongoing__desc: Legal proceedings is ongoing
```

```suggestion
    mark_as_legal_proceedings_ongoing_desc: Legal proceedings are ongoing
```
Two underscores intentional?
(or some other way to resolve "proceedings is" plural/singlular confusion 😁 )